gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/model/GbGradebookData.java

```diff
@@ -445,6 +451,7 @@ private String getString(final String key) {
 			studentDefinition.setLastName(student.getStudentLastName());
 			studentDefinition.setHasComments(formatColumnFlags(student, g -> StringUtils.isNotBlank(g.getGradeComment())));
 			studentDefinition.setHasDroppedScores(formatColumnFlags(student, g -> g.isDroppedFromCategoryScore()));
+			studentDefinition.setHasExcuse(formatExcuseData(student));
```

Comment:
You can just use formatColumnFlags(student, g -> g.isExcuse()) here and then you don't need formatExcuseData at all